administrators.rst

```diff
@@ -326,24 +293,37 @@ Fields:
 
 .. _dspace:
 
-DSpace via SWORD2 API or DSpace via REST API
+DSpace via SWORD2 API
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-DSpace locations are currently supported only for AIP Storage locations.  Becasue
-DSpace is typically used as a public-facing system, the behaviour is different
+DSpace via SWORD2 locations are currently supported only for AIP Storage locations.
+Because DSpace is typically used as a public-facing system, the behaviour is different
 than when using other access protocols for AIP Storage: upon deposit in DSpace,
-the AIP will be split into an objects bitstream, which contains all original and
-normalized objects in the AIP, as well as a metadata bitstream, which contains
-all of the bag artifacts, metadata and logs that one would expect to see in
-an Archivematica AIP. The metadata bitstream can optionally be restricted; see
-below. Presently, the Storage Service and Dashboard are not capable of
-downloading/reconstituting the AIP- this must be done manually from
-the DSpace interface.
-
-If using DSpace as the AIP location in conjunction with the ArchivesSpace workflow
-in the :ref:`Appraisal tab <archivematica:appraisal>`, a post Store AIP hook will
-send the DSpace handle to the ArchivesSpace digital object record upon AIP
-storage.
+the AIP will be split into two parts:
+
+* a compressed objects file (bitstream), which contains all original and
+  normalized objects
+* as well as a metadata file (bitstream), which contains all of the bag
+  artifacts, metadata and logs
+
+The metadata bitstream can optionally be restricted; see below. Presently, the
+Storage Service and Dashboard are not capable of downloading/reconstituting the
+AIP - this must be done manually from the DSpace interface.
+
+If using DSpace as the AIP location in conjunction with the ArchivesSpace
+workflow in the :ref:`Appraisal tab <archivematica:appraisal>`, a post Store AIP
+hook will send the DSpace handle to the ArchivesSpace digital object record upon
+AIP storage. The ArchivesSpace configuration is set up in the
+:ref:`Dashboard administration tab <archivematica:dashboard-admin>`.
+
+.. note::
+
+   Note that the DSpace via SWORD2 API space makes use of the DSpace REST API to
+   change the permissions of the metadata file. This means that you need to make
+   sure that the REST API is configured, see the `DSpace 5 REST API
+   documentation`_. The DSpace REST functionality only supports DSpace 5 and not
```

Comment:
Just noticed that the formatting looks a little weird in the GH UI for this link, does it work when you build HTML or did the linebreak break it?